index.html

```diff
@@ -216,16 +216,16 @@
 
     <a href="https://html.spec.whatwg.org/multipage/#the-iframe-element">
     <dfn>iframe</dfn></a>.
+
+    <a href="https://html.spec.whatwg.org/multipage/#settings-object">
+    <dfn>settings object</dfn></a>.
```

Comment:
"Settings object" is an internal slot of scripts. "Environment settings object" is the type.